### PR TITLE
JupyterLab Accessibility post 3

### PR DESCRIPTION
**Pull request by isabela-pf**
Another JupyterLab accessibility post that I offer as tribute to the deities that are my 15Five priorities for the past few months.

This one is about how accessibility needs to serve the many different and sometimes contradicting needs of disabled people and a reflection on the JupyterLab accessibility meeting that sparked that discussion.

I haven't made time to create a header image yet, but review is still welcome as well as a scheduled posting date if that's still the process. Deadlines always make me finish things. 🌻

### CODE REVIEW

**Comment by juanis2112** [comment]
Thanks @isabela-pf I will review this one. Is it okay to publish this one next Friday on September 10th? I have one from Mars coming up this Friday but if you need it up sooner, that's fine too :)

**Comment by isabela-pf** [comment]
That's fine with me! Just wanted to make sure it wasn't lost.

File: posts/2021/08/not-a-checklist.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+<!--
+.. title: Not a checklist: different accessibility needs in JupyterLab
+.. slug: not-a-checklist
+.. date: 2021-08-30 18:00:00 UTC-00:00
```
Comment (trallard) [inline posts/2021/08/not-a-checklist.md:4]: Will need to fix the date before merging

File: posts/2021/08/not-a-checklist.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+### What’s next?
+
+After several months of JupyterLab accessibility meetings and lovely comments 
+from people across open source communities (like our friends in [NumPy](https://numpy.org/) 
+and [Spyder](https://www.spyder-ide.org/)), people have repeatedly asked for 
+what my process is for evaluating accessibility in my regular work as an abled 
+person. Stay tuned for my personal recommendations and references for making 
+accessibility a constant consideration even when it is not a part of your 
+personal experience!
```
Comment (trallard) [inline posts/2021/08/not-a-checklist.md:124]: Now that the cat is out of the bag we can also add a note about the CZI grant and link to the previous post in the blog?

**Comment by trallard** [comment]
Love as oer usual the casual and conversational tone of the post. 
Though it seems it is missing another header or two somewhere to give for a pause or introduce actions/call to actions.

**Comment by Carreau** [comment]
> his one next Friday on September 10th

Just want to point out that Fridays are not the best days to get visibility on blog posts.

**Comment by juanis2112** [comment]
Thanks @Carreau. When do you think is a good date?

**Comment by Carreau** [comment]
> When do you think is a good date?

From the marketing folks, it's usually not the first of last day of the week, which usually is Tuesday/Wednesday/Thursday, unless you have holidays (at least for opinion based/technical things), it's not true if you want to be an influencer on instagram were friday is the best. Way to remember it: on monday, people start working on catchup on what happened during the weekend and so don't check social media, on friday they are looking forward to week-end and don't want to think.

Also Friday (US) is already later in the day for Europe and is even Saturday in Australia ...

 It's also really good to have consistency – so we can stick to friday if we decide to, as long as we stick to it.

As what we publish is _mostly_ technical then I would suggest to stick to Tue/Wed/Thu, but @juanis2112 you are the boss.

**Comment by isabela-pf** [comment]
@trallard thanks for the feedback! I added more headers and a grant mention.

I'll update the date when I get confirmation about what date I should update it to. Thanks everyone!

**Comment by juanis2112** [comment]
Thanks @Carreau. This is very good to know. I'm not an expert on marketing so all the information you have is welcomed. If @isabela-pf is okay with it, let's leave this one for next Tuesday (14th) and I'll try to stick to Tuesday for next posts.

File: posts/2021/09/not-a-checklist.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+and blog post (like this one!), I feel as though there is a lingering sense of 
+singular absoluteness that pervades the discussions around accessibility that 
+I encounter. Something that leaves abled folk like myself almost subconsciously 
+feeling , “if I just follow this this top ten accessibility misses as listed 
+here, I will have succeeded in making a perfectly accessible piece of software 
+and never have to worry about it again.” Once again, this notion sounds just as 
+silly to me when stated explicitly, and yet I find myself falling into that 
+trap after reading every listicle. 
+
+Most of all,  is it any surprise that this might be what happens when disabled
```
Comment (juanis2112) [inline posts/2021/09/not-a-checklist.md:41]:
```suggestion
Most of all, is it any surprise that this might be what happens when disabled 
```
Double space

**Review by juanis2112 — APPROVED** [review]
Found one extra space but otherwise looks fine. Amazing blogpost! I think this was very interesting @isabela-pf. Thanks for taking the time to write this. I'll definitely stay tuned. This is ready to merge

**Comment by rgommers** [comment]
Nice post! One small issue, it is missing the `<!-- TEASER_END -->` marker, so it takes up too much space on https://labs.quansight.org/blog/. Could one of you open a follow-up PR with a fix?

**Comment by isabela-pf** [comment]
Thanks for catching that! #253 should fix it.

**Pull request merged**